### ISSUE TITLE
Question about Saga

### ISSUE THREAD

**Issue by ayazzali** [ebubekirdinc/SuuCat#9]
What if while creating order rabbit will be rebooting or smth - so that means  OrderCreatedEvent will not be created created and order will always stay at Pending status.

PS just learning sagas and stuff. Your repo is excellent for learning. Thanks
PPS dint see some simple ui so we can easily create user product and order it

**Comment by ebubekirdinc** [comment]
To make sure it is sent, the Outbox Pattern must be applied to send "at least once". I will use it in this project to show the outbox pattern in the future.
You can use Swagger UI to send requests. I also plan to share Postman collection when it matures. 

Thanks.

**Comment by elenacalugaru** [comment]
Hello,
I have a question, for eg when OrderCompletedEvent is published by the statemachine to both Order and Notification services shall consume the same event, but if Order consumes first the message then it wont be consumed by Notification because once it is consumed it is deleted from the queue.

**Comment by ebubekirdinc** [comment]
Yes, in this case it should be sent as an event, not as a command.

**Comment by akash-karve-dev** [comment]
> To make sure it is sent, the Outbox Pattern must be applied to send "at least once". I will use it in this project to show the outbox pattern in the future. You can use Swagger UI to send requests. I also plan to share Postman collection when it matures.
> 
> Thanks.

I am eagerly waiting for outbox/inbox pattern. I liked your saga implementation

**Comment by ebubekirdinc** [comment]
> > To make sure it is sent, the Outbox Pattern must be applied to send "at least once". I will use it in this project to show the outbox pattern in the future. You can use Swagger UI to send requests. I also plan to share Postman collection when it matures.
> > Thanks.
> 
> I am eagerly waiting for outbox/inbox pattern. I liked your saga implementation

Thanks. I will develop the inbox/outbox pattern soon.

**Comment by Misiu** [comment]
Hi there, any update on the inbox/outbox pattern?
I'm currently learning MT with Sagas and your repository is great place for learning :)
Thank you for sharing the code 👍👏💪
